Dockerfile

```
@@ -19,6 +19,6 @@ COPY ./.nginx/nginx.conf /etc/nginx/nginx.conf
 RUN rm -rf /usr/share/nginx/html/*
 
 # Copy from the stahg 1
-COPY --from=builder /login-frontend/dist /usr/share/nginx/html
+COPY --from=builder /vue-ui/dist /usr/share/nginx/html
 
-EXPOSE 8000
+EXPOSE 80/tcp
```

add a newline at the end of the file
```suggestion
EXPOSE 80/tcp

```